content/en/docs/tasks/configure-pod-container/enforce-standards-admission-controller.md

```diff
@@ -4,19 +4,17 @@ reviewers:
 - tallclair
 - liggitt
 content_type: task
-min-kubernetes-server-version: v1.22
+min-kubernetes-server-version: v1.25
```

Comment:
Why this change?

Reply:
Pod Security Admission is GA in Kubernetes 1.25

Reply:
This page already indicates 1.25 is required for the v1 config, and links users to instructions that let them try/use this feature in alpha and beta state in 1.22, 1.23 and 1.24. What does this bit of the configuration do, and will it be misleading to label the whole page as being applicable only to 1.25+?

Reply:
I tweaked the page elsewhere.

Have a look at https://deploy-preview-37733--kubernetes-io-vnext-staging.netlify.app/docs/tasks/configure-pod-container/enforce-standards-admission-controller/#before-you-begin and https://deploy-preview-37733--kubernetes-io-vnext-staging.netlify.app/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/#before-you-begin to see how these look now.


I think we should set `min-kubernetes-server-version` to the GA version. I can see why some people would prefer to set it to the version where the feature went beta / became enabled by default. Personally, I prefer to recommend the GA feature and then have a narrative explanation about what to do if your cluster is older.

Reply:
The `min-kubernetes-server-version` influences how a `{{% version-check %}}` shortcode renders.

Reply:
I've revised again, a preview should already be available.